### DISCUSSION TITLE
Without opening GUI, how to link 1.wbjn, 2.py and 3.py by some command (script) so that I can run one by one and get data for updated geometry each time.

### DISCUSSION THREAD

**Discussion by Owen6136** [ansys/pyansys#628] [General]
Kind help!

I went through your input and I need your further help. I specify my question as follow:
1.	I have WBJN file (1.wbjn), I can modify/set material properties 
2.	Geometry file (2.py), I can change some geometry by some parameters 
3.	Mechanical (3.py): mesh setting for updated geometry. Define boundary, solve and data output

Without opening GUI, how to link 1.wbjn, 2.py and 3.py by some command (script) so that I can run one by one and get data for updated geometry each time. 

![Workbench](https://github.com/user-attachments/assets/9c017ccb-a999-437a-8231-64421b5a86f5)

**Comment by RobPasMue** [comment]
Hi @Owen6136 - are you using any specific PyAnsys library like PyWorkbench?
